src/Sylius/Bundle/AdminBundle/Twig/Component/Shipment/ShipComponent.php

```diff
@@ -22,8 +22,8 @@
 use Symfony\UX\LiveComponent\ComponentWithFormTrait;
 use Symfony\UX\LiveComponent\DefaultActionTrait;
 
-#[AsLiveComponent(name: 'SyliusAdmin.Shipment.ShipType', template: '@SyliusAdmin/shipment/component/ship.html.twig')]
-final class ShipType
+#[AsLiveComponent]
+final class ShipComponent
```

Comment:
```suggestion
final class ShipFormComponent
```
Just to emphasize that it is a form.